fly/integration/login_insecure_test.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 var _ = Describe("login -k Command", func() {
-	var loginATCServer *ghttp.Server
+	var loginAtcServer *ghttp.Server
```

Comment:
Small nit can we change this back to loginATCServer (capital ATC) to minimize the diff